Src/WitsmlExplorer.Frontend/services/logObjectService.tsx

```diff
@@ -1,36 +1,70 @@
 import { ErrorDetails } from "models/errorDetails";
-import { LogData } from "models/logData";
+import { LogData, LogDataRequestQuery } from "../models/logData";
 import { ApiClient } from "services/apiClient";
 import AuthorizationService from "services/authorizationService";
 import NotificationService from "services/notificationService";
+import LogCurveInfo from "../models/logCurveInfo";
 
 export default class LogObjectService {
   public static async getLogData(
     wellUid: string,
     wellboreUid: string,
-    logUid: string,
-    mnemonics: string[],
+    requestData: LogDataRequestQuery[],
     startIndexIsInclusive: boolean,
     startIndex: string,
     endIndex: string,
     loadAllData: boolean,
     abortSignal: AbortSignal
   ): Promise<LogData> {
-    if (mnemonics.length === 0) return;
+    if (requestData.length === 0) return;
     const params = [
+      `startIndex=${encodeURIComponent(startIndex)}`,
+      `endIndex=${encodeURIComponent(endIndex)}`,
```

Comment:
What happends if startIndex or endIndex is falsy? Earlier, it did not get added, but now you potentially set startIndex=null or startIndex=